app/src/sharedTest/java/org/oppia/app/testing/FontScaleTestActivityTest.kt

```diff
@@ -0,0 +1,93 @@
+package org.oppia.app.testing
+
+import android.content.Context
+import android.content.Intent
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.model.StoryTextSize
+import org.oppia.app.utility.FontSizeMatcher
+
+/** Tests for [FontScaleTestActivity]. */
+@RunWith(AndroidJUnit4::class)
+class FontScaleTestActivityTest {
+  lateinit var context: Context
+  @Before
+  fun setUp() {
+    Intents.init()
+    context = ApplicationProvider.getApplicationContext()
+  }
+
+  @After
+  fun tearDown() {
+    Intents.release()
+  }
+
+  private fun createHomeActivityIntent(storyTextSize: String): Intent {
+    return FontScaleTestActivity.createFontScaleTestActivity(
+      context,
+      storyTextSize
+    )
+  }
+
+  @Test
+  fun testFontScaleTestActivity_smallTextSizeIsVerifiedSucessfully() {
```

Comment:
Typo in the word `Sucessfully` -> `Successfully`. Here and everywhere else.

Reply:
updated all method name typo errors